### PR TITLE
Feature/es ulist

### PR DESCRIPTION
**Pull request by SanthoshRaju91**
This is first feature complete on es-ulist completion, UI looks like in the images attached. Please review and comment if something needs to be changed or added.

screen shot 2018-02-25 at 3 39 17 pm
screen shot 2018-02-25 at 3 39 23 pm
screen shot 2018-02-25 at 3 39 31 pm

### CODE REVIEW

**Comment by rwwagner90** [comment]
Should we do something to ensure those images are not stretched out?

**Comment by SanthoshRaju91** [comment]
@rwwagner90 css property `object-fit: cover` worked. Here is the screenshot of the latest and it looks like.

screen shot 2018-03-02 at 9 05 41 am

**Comment by rwwagner90** [comment]
Thanks for fixing that @SanthoshRaju91! I will try to get this reviewed this weekend.

File: addon/templates/components/es-ulist.hbs [hunk 1]
```diff
@@ -1 +1,11 @@
-{{yield}}
+<div>
+    <ul aria-describedby="{{ariaDescribedby}}">
+        {{es-heading headingText=title tagName="h3" class="title"}}
```
Comment (MelSumner) [inline addon/templates/components/es-ulist.hbs]: This won't be valid HTML- the h3 has to go outside of the `<ul>` element.

Comment (SanthoshRaju91) [reply addon/templates/components/es-ulist.hbs]: Fixed the review comment and updated the PR. Thank you

**Comment by MelSumner** [comment]
@SanthoshRaju91 thank you for your patience- it will be a few more days before I can look at this. In the meantime, would you perhaps be able to see why the build is failing? Are there some tests that need to be refactored due to the code changes?

**Comment by rwwagner90** [comment]
@MelSumner I think Travis just messed up here. I restarted those jobs and hoping for the best! I'm also not really able to dedicate much review time until after EmberConf, but I will get back into this a bunch after that.

**Comment by SanthoshRaju91** [comment]
@MelSumner I understand, good luck for your session. This is failing intermittently, as the travis build is not able to resolve the eslint.

File: addon/components/es-ulist.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+import { computed } from '@ember/object';
 
 export default Component.extend({
   layout,
   classNames: ['es-ulist'],
 
   //accessibility support
-  ariaDescribedby: null,
-  ariaLabel: null,
+  ariaDescribedby: computed.alias('list.id'),
```
Comment (MelSumner) [inline addon/components/es-ulist.js]: Since this is the list, ariaDescribedBy will be
- possibly null
- the value of the heading ID

Comment (SanthoshRaju91) [reply addon/components/es-ulist.js]: Yes @MelSumner. Should I need to make it default.

**Comment by MelSumner** [comment]
@SanthoshRaju91 thank you for your contribution! :+1:

**Comment by SanthoshRaju91** [comment]
@MelSumner @rwwagner90 thank you so much for reviewing and mergning my PR. I understand the pattern of ur code review, I will pick up some more components to build.

**Comment by rwwagner90** [comment]
We really appreciate your work on this @SanthoshRaju91. Nice job! 🎉

**Pull request merged**